ee/clickhouse/sql/events.py

```diff
@@ -114,6 +123,46 @@
 FROM events
 """
 
+EVENTS_WITH_DENORMALIZED_PROPS = """
+CREATE TABLE {table_name}_with_denormalized_props
+(
+    uuid UUID,
+    event VARCHAR,
+    properties VARCHAR,
+    timestamp DateTime64(6, 'UTC'),
+    team_id Int64,
+    distinct_id VARCHAR,
+    elements_chain VARCHAR,
+    created_at DateTime64(6, 'UTC'),
+    properties_test_prop VARCHAR
```

Comment:
Is the idea here that we're manually going to rebuild the tables each time we want to put a property into a column?

Reply:
Yep annoyingly I think that's the only way to do this, unless I'm wrong @fuziontech

Reply:
Nope, that's what we have to do 😢

Reply:
well done here btw - this is perfect

Reply:
so I was doing a bit of research and found this little nugget that will make our lives _much_ easier
https://image.slidesharecdn.com/lfgouwaqrbav2qmz0z6a-signature-1b0c613dee020c35bf055baae1ebcab6567363fcb79a63f24e04d0a620019a60-poli-190816134217/95/tricks-every-clickhouse-designer-should-know-by-robert-hodges-altinity-ceo-22-638.jpg?cb=1565963144